src/jobs/publish_s3.yml

```diff
@@ -0,0 +1,35 @@
+description: Publish files to S3
+
+executor: aws
+
+parameters:
+  public:
+    type: boolean
+    description: Whether to store in the public bucket
+  path:
+    type: string
+    description: Path to store the files in S3
+  files:
+    type: string
+    description: Files to publish separated by space
+
+steps:
+  - attach_workspace:
+      at: ~/workdir
+
+  - run_aws:
+      cmd_name: Send files to S3
+      cmd: |
+        if [[ << parameters.public >> ]]; then
+          bucket=$AWS_PUBLIC_ARTIFACTS_BUCKET
+        else
+          bucket=$AWS_PRIVATE_ARTIFACTS_BUCKET
+        fi
+
+        for file in << parameters.files >> ; do
+          filename=$(basename $file)
+          AWS_PROFILE=production aws s3 cp $file $bucket/<< parameters.path >>/$(cat .version)/$filename
+        done
+        if $(grep -q -E '^[0-9]+\.[0-9]+\.[0-9]+$' .version); then
```

Comment:
what is the aim of this if? Which cases it catches, what are the cases that it ignores stuff?

Reply:
This if is to avoid setting the latest to a feature branch in case someone tries to publish to s3 from a feature branch.
This orb is tightly coupled with the semantics of git-version.